ast/ast.go

```diff
@@ -1489,6 +1495,39 @@ type AlterTable struct {
 	TableAlternation TableAlternation
 }
 
+// AlterChangeStream is ALTER CHANGE STREAM statement node.
+//
+// ALTER CHANGE STREAM {{.Name | sql}}
+// {{ if or len(.Watch) > 0 .WatchAll }}
+// SET FOR {{ .WatchAll }}ALL{{ else }}{{ .Watch | sqlJoin ","}}{{ end }}
+// {{ end }}
+// {{ if  .DropAll }}
+// DROP FOR ALL
+// {{ end }}
+// {{ if len(.Options) > 0 }}
+// SET OPTIONS {{ .Options | sqlJoin ","}}
+// {{ end }}
+type AlterChangeStream struct {
+	// pos = Alter
+	// end = {{ if .Options }}
+	//          .Options.end
+	//       {{ else }}
+	//         {{ if len(.Watch) > 0 }}
+	//           .Watch[len(.Watch)-1].end
+	//         {{ else if .WatchAll }}
+	//           "ALL".end
+	//         {{ else }}
+	//           .Name.end
+	//         {{ end }}
+	//       {{ end }}
```

Comment:
We have not used Go's template-like syntax for `pos` and `end`. If you want to use some rich features here, I think they are code smells.

Reply:
I just would like to write a comment for description of pos or end like the existing code comments,
What is the syntax as in https://github.com/cloudspannerecosystem/memefish/blob/86cbd88693043a3c1ed99d69dbcdd814b187a985/ast/ast.go#L385-L386 ?